src/configuration/configuration-base.ts

```diff
@@ -45,32 +50,32 @@ export default class Configuration {
         DEBUG_LOGGER(error);
     }
 
-    static _resolveFilePath (path) {
+    private static _resolveFilePath (path: string | null): string | null {
         if (!path)
             return null;
 
         return isAbsolute(path) ? path : resolvePathRelativelyCwd(path);
     }
 
-    async init () {
+    public async init (): Promise<void> {
     }
 
-    mergeOptions (options) {
+    public mergeOptions (options: object): void {
         Object.entries(options).map(([key, value]) => {
-            const option = this._ensureOption(key, value, optionSource.input);
+            const option = this._ensureOption(key, value, OptionSource.Input);
 
             if (value === void 0)
                 return;
 
             if (option.value !== value &&
-                option.source === optionSource.configuration)
-                this._overridenOptions.push(key);
+                option.source === OptionSource.Configuration)
+                this._overriddenOptions.push(key);
 
             this._setOptionValue(option, value);
         });
     }
 
-    getOption (key) {
+    public getOption (key: string): OptionValue {
```

Comment:
Why we need the `public` keyword?

Reply:
This method is used in other modules.
https://github.com/DevExpress/testcafe/blob/9b08199af5841f65540846a186c8d19c78b07d9d/src/index.js#L54